src/govuk/tools/_rebrand.scss

```diff
@@ -0,0 +1,63 @@
+////
+/// @group tools
+////
+
+/// Wraps rebranded properties in the feature flag selector
```

Comment:
We're getting warnings in the console during tests (on both v5 and v4):

`Warning: G] @parameter must at least have a name. Location: _rebrand.scss:5:28`

Not critical, but would be good to figure out how to appease the sass gods and silence the warnings.

Reply:
The params shouldn't have square brackets around their names:

```scss
/// @param {String} $property - The name of the property being rebranded
```

vs 

```scss
/// @param {String} [$property] - The name of the property being rebranded
```

Square brackets are [used to define what the default value of the property is](http://sassdoc.com/annotations/#parameter), so sassdoc is seeing the parameter name as missing.

Reply:
Nice catch! This'll need updating on v5 as well.

Reply:
I've sorted this on this branch and I'll do a separate one for v5 in a jiffy